### PR TITLE
Add callbacks parameter to OperationTrait

### PR DESCRIPTION
**Pull request by edipoReboucas**
"callbacks" parameter is missing on OperationTrait constructor.

I implemented this bugfix based on PR https://github.com/zircote/swagger-php/pull/1191.

### CODE REVIEW

**Review by DerManoMann — CHANGES_REQUESTED** [review]
Couple nits but otherwise looking good.
Very happy about the updated tests :)

File: src/Attributes/OperationTrait.php [hunk 1]
```diff
@@ -16,6 +16,7 @@ trait OperationTrait
      * @param string[]                  $tags
      * @param Parameter[]               $parameters
      * @param Response[]                $responses
+     * @param array                     $callbacks
```
[the remark this answers is not in the document]
Comment (edipoReboucas) [reply src/Attributes/OperationTrait.php]: Done

File: tests/Analysers/ReflectionAnalyserTest.php [hunk 2]
```diff
@@ -102,6 +102,7 @@ public function testApiDocBlockBasic(AnalyserInterface $analyser): void
         $this->assertIsArray($operations);
 
         $spec = $this->fixture('Apis/basic.yaml');
+
```
[the remark this answers is not in the document]
Comment (edipoReboucas) [reply tests/Analysers/ReflectionAnalyserTest.php]: It's a typo, fixed.

**Review by DerManoMann — APPROVED** [review]
Gooking good - thanks.

**Comment by DerManoMann** [comment]
Thanks @edipoReboucas

**Pull request merged**